CONTRIBUTING.md

```diff
@@ -1 +1,3 @@
+Before opening a pull request, try running `yarn generate:release-notes`!
```

Comment:
nit: might be nice to keep this comment in the docs site only, so that we have just one source of truth for contributions!